### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Pratik-TWS**
#### PR

- [] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [] My code has been well-documented (function documentation, inline comments,
  etc.)

#### Public Documentation Updates

- [] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [] OS: 
- [] Node version: 
- [] Browser:
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Review by Pratik-TWS — COMMENTED** [review]
Reviewed

File: extensions/default/src/DicomWebDataSource/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { api } from 'dicomweb-client';
 import { DicomMetadataStore, IWebApiDataSource, utils, errorHandler, classes } from '@ohif/core';
-
+//Pratik
```
Comment (rajangoswamiTWS) [inline extensions/default/src/DicomWebDataSource/index.js]: Remove this line

File: extensions/default/src/DicomWebDataSource/index.js [hunk 2]
```diff
@@ -381,12 +381,11 @@ function createDicomWebApi(dicomWebConfig, servicesManager) {
        */
       const addRetrieveBulkData = instance => {
         const naturalized = naturalizeDataset(instance);
-
+        // Pratik
```
Comment (rajangoswamiTWS) [inline extensions/default/src/DicomWebDataSource/index.js]: Remove this line. ALso if same comments are there elsewhere, please take care of those.

File: extensions/default/src/DicomWebDataSource/index.js (hunk 1 above)
Comment (Pratik-TWS) [reply extensions/default/src/DicomWebDataSource/index.js]: Resolve

File: extensions/default/src/DicomWebDataSource/index.js (hunk 2 above)
Comment (Pratik-TWS) [reply extensions/default/src/DicomWebDataSource/index.js]: Resolve

**Review by rajangoswamiTWS — APPROVED** [review]

**Pull request merged**